README.md

```diff
@@ -39,19 +39,14 @@ With the ESnext field declarations proposal, the above example can be written as
 class Counter extends HTMLElement {
```

Comment:
They end up behaving the same, but they aren't equivalent. Using functions on instances (with class properties or not) in React components is an antipattern, just not one the React team has decided is an antipattern yet :-)